core/chains/evm/client/errors.go

```diff
@@ -88,7 +88,7 @@ var geth = ClientErrors{
 	NonceTooLow:                       regexp.MustCompile(`(: |^)nonce too low$`),
 	NonceTooHigh:                      regexp.MustCompile(`(: |^)nonce too high$`),
 	ReplacementTransactionUnderpriced: regexp.MustCompile(`(: |^)replacement transaction underpriced$`),
-	TransactionAlreadyInMempool:       regexp.MustCompile(`(: |^)(?i)(known transaction|already known)`),
+	TransactionAlreadyInMempool:       regexp.MustCompile(`(?i)(known transaction|already known)`),
```

Comment:
Why here instead of `arbitrum` on line 134?

Reply:
Tenderly could be doing this for other errors too, which we may not have encountered yet?

Reply:
Many people use this file as an index, so I'd also suggest to move this under `arbitrum`. If it's being used by others but we don't know it's ok, but at least let's be consistent with what we know so far.